Assets/_Game/Scripts/Core/Data/Enums/GameDevice.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+
+namespace Ibit.Core.Data.Enums
+{
+    public enum GameDevice
+    {
+        [Description("Pitaco")]
+        Pitaco,
+        [Description("Manovacuometro")]
+        Manovacuometro,
```

Comment:
Alterei os nomes aqui pra "Manovacuometro", com "o", é a nomenclatura que estou usando no jogo e a que encontrei maior número de incidências na pesquisa do Google.
Nem sei se isso irá gerar algum bug, se for, retorne por favor.

Reply:
Feito!